pocs/mount/serial.py

```diff
@@ -26,6 +28,10 @@ def __init__(self, *args, **kwargs):
             self.serial = rs232.SerialData(port=self._port, baudrate=9600)
         except Exception as err:
             self.serial = None
+            # This won't be triggered because SerialData is created even if the mount port
+            # can't be opened. We can't tell the difference between not currently present
+            # and totally invalid (e.g. com1: on a Linux box, or /dev/tty7 on Windows),
+            # TODO(wtgee): What would you like to do about this?
```

Comment:
This is why I was a little skeptical about creating the `SerialData` object regardless. The `error.BadSerialConnection` is a good replacement, but it is a big annoying when in use via `pocs_shell`, because the error right here will trigger before anything else is set up and tell the user to use a simulator. 

However, I think that could also be handled in a better manner. I think we could simple remove the `try/except` here.

Reply:
I've removed that (and will send another PR).

I'll create an issue to discuss the issue you raise.